<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSettingDlg</class>
 <widget class="QDialog" name="DataSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>162</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>光伏预测设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  *[text=&quot;确定&quot;] {
	background: rgb(26, 115, 232);
	color: rgb(255, 255, 255);
	font: 12pt &quot;PingFang SC&quot;;
	border: 0px;
	border-radius: 10%;
	padding-left: 25px;
	padding-right: 25px;
	border-width: 1px;
	border-color: rgb(100, 100, 100);
	border-style: solid;
	border-radius: 10%;
}

*[text=&quot;确定&quot;]:hover {
	background: rgb(52, 160, 255);
	font: 12pt &quot;PingFang SC&quot;;
	border: 0px;
	border-radius: 10%;
}

 *[text=&quot;取消&quot;] {
	background: rgb(229, 229, 229);
	font: 12pt &quot;PingFang SC&quot;;
	border-width: 1px;
	border-color: rgb(100, 100, 100);
	border-style: solid;
	border-radius: 10%;
	padding-left: 25px;
	padding-right: 25px;
}

 *[text=&quot;取消&quot;]:hover {
	background:rgb(255, 255, 255);
	font: 12pt &quot;PingFang SC&quot;;
	border-style: solid;	
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Label">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>15</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 700 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>命名空间：</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameSpaceEdit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;给你的数据集命名&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 500 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">
font: 700 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>训练时间尺度：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="seqLenSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PingFang SC'; font-size:15pt; font-weight:500; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;尽量是2的倍数&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 500 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>8</number>
        </property>
        <property name="value">
         <number>96</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">
font: 700 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>预测长度：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="predictLenSpinBox">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 500 15pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>180</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
